g_docs.go

```diff
@@ -1216,3 +1213,24 @@ func generatePathInfo(file *ast.File) (map[string]string, error) {
 
 	return pathInfo, nil
 }
+
+func getPackageName() (string, error) {
+	// Go modules :)
+	f, err := os.ReadFile(modfileName)
+	if err == nil {
+		mf, err := modfile.Parse(modfileName, f, nil)
+		if err == nil {
+			return mf.Module.Mod.Path, nil
+		}
+	}
+
+	// Gopath :(
+	currentPath, err := os.Getwd()
+	if err != nil {
+		return "", err
+	}
+
+	goSrcPath := os.Getenv("GOPATH") + "/src/"
```

Comment:
(nit) should we just use `Source` or capitalize it to be `SRC`?
```suggestion
	goSourcePath := os.Getenv("GOPATH") + "/src/"
```

```suggestion
	goSRCPath := os.Getenv("GOPATH") + "/src/"
```

Reply:
If you think it's necessary I'll be happy to do so. But is this our default approach when working in legacy software?

If I had come into this task with that mindset, I think I would have changed every other line in this file g_doc.go. Look at all the var and function names, the misspellings, the imprecise nomenclature (e.g. i'm assigning the return value of getPackageName() to **basePath** for Pete's sake...). So you can probably imagine I was triggered the whole time working on this task, but I refrained from changing anything other than the truly necessary (ie things that affect readability). You can already tell I drew the line at banner.go.

Anyhoo, I find this ultra-surgical approach the appropriate way to do it in legacy code. Wdyt?

Reply:
hmm no not necessary, but I was thinking since you're creating a **new** function here, why don't we start addressing the variable name only for this function. it's not going to affect older codes and the changes would not be significant. Just like what we did in Doraemon, no? when we want to create a new module we put in `pkg` instead of `module`, and then we use the standard of what we think is the best in `pkg`.

it's not going to make the whole bee codebase to be readable, but at least for the newer functions, dev would have a better time reading it. wdyt?

Reply:
Yeah your logic is sound. Will update.

Reply:
Addressed in f59bd0a.